build/build.go

```diff
@@ -54,9 +54,9 @@ var (
 const (
 	//nolint:gosec // G101: false-positive
 	printFallbackImage = "docker/dockerfile:1.5@sha256:dbbd5e059e8a07ff7ea6233b213b36aa516b4c53c645f1817a4dd18b83cbea56"
-	// https://github.com/moby/buildkit/commit/3fd813cfa5eec6633b21da8164e93673d005df02
+	// https://github.com/moby/buildkit/commit/71f99c52a669dc0322b5ea57bc28a09c20427227
 	//nolint:gosec // G101: false-positive
-	printLintFallbackImage = "docker/dockerfile-upstream@sha256:2e302d1450bb1c3d9ef9b52800d2f3738cdfd21ead73a403e32a9bc275290e7c"
+	printLintFallbackImage = "docker.io/docker/dockerfile-upstream:master@sha256:47663570b6cc49ed90dc6e3215090a366989ab934d12dc93856a8ae0d27a95e7"
```

Comment:
I see `docker.io` prepend, should we do the same here https://github.com/docker/buildx/pull/2417/commits/810d36ff129b4ee74dd589549ed6a38a979d2a90#diff-7fdd319c657fbde54b83a5dfa233c5c2c417b291739ff75c05dc5d83ee7a9d24R56 and https://github.com/docker/buildx/blob/1b16594f4aaa822e98436ecaad80d5e625ea071c/driver/bkimage/bkimage.go#L4-L5?

Reply:
Functionally it doesn't matter